breathecode/admissions/tests/urls/tests_academy_cohort.py

```diff
@@ -1580,3 +1580,33 @@ def test_academy_cohort_with_data_testing_cache_and_remove_in_post(self):
 
         self.test_academy_cohort__with_data(base)
         self.assertEqual(self.cache.keys(), cache_keys)
+
+    @patch(GOOGLE_CLOUD_PATH['client'], apply_google_cloud_client_mock())
+    @patch(GOOGLE_CLOUD_PATH['bucket'], apply_google_cloud_bucket_mock())
+    @patch(GOOGLE_CLOUD_PATH['blob'], apply_google_cloud_blob_mock())
+    def test_academy_cohort_id__delete__cohort_with_students(self):
+        self.headers(academy=1)
+        model = self.generate_models(authenticate=True, profile_academy=True, 
+            capability='crud_cohort', role='potato' ,cohort_user=True)
+        url = reverse_lazy('admissions:academy_cohort_id', kwargs={'cohort_id': 1})
```

Comment:
This test is correct, but is need be moved to `tests_academy_cohort_id.py` because its called using `academy/cohort/<cohort_id:str>`